content/academy/anti_scraping.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Anti-Scraping Protection
```

Comment:
Please, use sentence case for titles, both in metadata and throughout the content headings. It'll add consistency with docs content and previous academy content, and generally looks more readable

<img width="314" alt="Screenshot 2022-04-08 at 09 17 20" src="https://user-images.githubusercontent.com/38246758/162384257-7dda0785-7c45-46c1-8930-4999c72556fa.png">